region/norfair/west/Crocomire Escape.json

```diff
@@ -549,7 +796,14 @@
         "canSpringBallJumpMidAir",
         "canUnmorphBombBoost",
         "h_canCrouchJumpDownGrab",
-        {"heatFrames": 800}
+        {"heatFrames": 800},
+        {"or": [
+          "canInsaneJump",
+          {"and": [
+            {"heatFrames": 250},
+            "h_additionalBomb"
+          ]}
```

Comment:
I added a leniency PB attempt in Expert. I could remove it or increase it; whatever you think is best.